gpytorch/module.py

```diff
@@ -87,7 +87,11 @@ def initialize(self, **kwargs):
                 val = float(val)
             if "." in name:
                 module, name = self._get_module_and_name(name)
-                module.initialize(**{name: val})
+                if isinstance(module, nn.ModuleList):
+                    idx, name = name.split(".", 1)
+                    module[int(idx)].initialize(**{name: val})
+                else:
+                    module.initialize(**{name: val})
```

Comment:
why not just 
```suggestion
                    module[int(idx)].initialize(name=val)
                else:
                    module.initialize(name=val)
```

Reply:
This treats "name" as the key in the kwarg vs the string the variable `name` contains.

Reply:
argh brainfart